pydle/features/rfc1459/client.py

```diff
@@ -325,7 +324,7 @@ async def quit(self, message=None):
             message = self.DEFAULT_QUIT_MESSAGE
 
         await self.rawmsg('QUIT', message)
-        await self.disconnect(expected=True)
+        await self.disconnect()
```

Comment:
why was this line changed?

Reply:
expected=True is the default argument of the disconnect() method. As such, there's no point in actually sending the value itself. This does not change functionality, simply following stylistic guidelines.

Reply:
It more clearly conveys intent, even if default.